aws_lambda_builders/validate.py

```diff
@@ -62,10 +63,11 @@ def validate_runtime(cls, required_language, required_runtime):
             p = subprocess.Popen(cmd,
                                  cwd=os.getcwd(),
                                  stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            p.communicate()
+            found_runtime, _ = p.communicate()
             if p.returncode != 0:
                 raise MisMatchRuntimeError(language=required_language,
-                                           required_runtime=required_runtime)
+                                           required_runtime=required_runtime,
+                                           found_runtime=str(found_runtime.decode('utf-8')))
```

Comment:
Is the decode valid in python2.7?

Reply:
checked it with an integ test, and ran it under python2.

under python2, even if its already string and we decode. we get to unicode.

```
Python 2.7.10 (default, Feb  7 2017, 00:08:15)
Type "copyright", "credits" or "license" for more information.

IPython 5.8.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: mystring="python"

In [2]: type(mystring)
Out[2]: str

In [3]: mystring.decode('utf-8')
Out[3]: u'python'
```

Reply:
perfect